applications/ecallmgr/src/ecallmgr_call_command.erl

```diff
@@ -1192,9 +1192,12 @@ execute_exten_handle_ccvs(DP, _Node, UUID, JObj) ->
         'false' ->
             ChannelVars = wh_json:to_proplist(CCVs),
             [{"application", <<"set ", (ecallmgr_util:get_fs_kv(K, V, UUID))/binary>>}
-             || {K, V} <- ChannelVars] ++ DP
+             || {K, V} <- ChannelVars, not execute_exten_is_original_ccv(K)] ++ DP
     end.
 
+execute_exten_is_original_ccv(<<?CHANNEL_VARS_EXT, _/binary>>) -> 'true';
```

Comment:
@jamesaimonetti i didn't add it because all other `execute_exte` don't have specs and this is just to prevent the loop.  all these `execute_exte` should actually be deleted in 4.0